docs/source/pre_application/testnet.rst

```diff
@@ -13,3 +13,25 @@ You can view it on `etherscan <https://rinkeby.etherscan.io/address/0xc3871e2c11
 .. attention::
 
     This testnet is currently being migrated to Threshold Network.
```

Comment:
Should we remove this note?

Reply:
I guess the ultimate "migration" to the Threshold Network is to have it supported on the Threshold Dashboard, but Ibex Rinkeby won't be supported on the Threshold Dashboard (but `oryx` will) so I'm not sure what is left to do to "migrate" Ibex.

Is our eventual goal to push staking providers onto `oryx`, but have internal development done on `ibex`? Right now, both are catered to in `ibex` Quoting from the testnet page:
```
NuCypher provides a public testnet running on the Ethereum Rinkeby testnet meant for staking providers and node operators to learn how to set up a PRE application node, as well as for internal development purposes.
```

Of course, this resolution does not necessarily have to be made in this PR and can be addressed in a separate issue/PR once it becomes more clear (if it isn't clear already).